### PR TITLE
Unregister service worker to bust the cache

### PR DESCRIPTION
**Pull request by mdomasevicius**
Signed-off-by: Mantas Domaševičius <mantas@mysterium.network>

### CODE REVIEW

File: src/serviceWorker.js [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+const isLocalhost = Boolean(
+    window.location.hostname === 'localhost' ||
+        // [::1] is the IPv6 localhost address.
+        window.location.hostname === '[::1]' ||
+        // 127.0.0.0/8 are considered localhost for IPv4.
+        window.location.hostname.match(/^127(?:\.(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)){3}$/),
+);
+
+export function register(config) {
+    if (process.env.NODE_ENV === 'production' && 'serviceWorker' in navigator) {
```
Comment (ignasbernotas) [inline src/serviceWorker.js:26]: is `process.env.NODE_ENV` available during runtime?

File: src/serviceWorker.js [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        const publicUrl = new URL(process.env.PUBLIC_URL, window.location.href);
+        if (publicUrl.origin !== window.location.origin) {
+            // Our service worker won't work if PUBLIC_URL is on a different origin
+            // from what our page is served on. This might happen if a CDN is used to
+            // serve assets; see https://github.com/facebook/create-react-app/issues/2374
+            return;
+        }
+
+        window.addEventListener('load', () => {
+            const swUrl = `${process.env.PUBLIC_URL}/service-worker.js`;
```
Comment (ignasbernotas) [inline src/serviceWorker.js:37]: Does this file exist somewhere?

Comment (mdomasevicius) [reply src/serviceWorker.js:37]: it is bundled afaik, this is CRA part.

File: src/serviceWorker.js (hunk 1 above)
Comment (mdomasevicius) [reply src/serviceWorker.js:26]: I would assume so, by default when you create new CRA project, this function gets called by default in index.tsx

**Review by ignasbernotas — APPROVED** [review]

**Pull request merged**